internal/errors/errors.go

```diff
@@ -0,0 +1,98 @@
+package errors
```

Comment:
How does this package differ from jettison's errors?

Reply:
To use jettison errors you need to import jettison which is everything. They main difference is that it is a standalone package and doesnt take snapshots of the stack traces. This error package is not meant to be used outside workflow so it means that stack traces wont be added from within in workflow but can be added inside the adapter and by the caller.